src/pyne/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 message("")
 
-set(SRC_FILES pyne.cpp)
-set(PUB_HEADERS pyne.h)
+execute_process(COMMAND amalgamate_pyne.sh)
+
+file(GLOB SRC_FILES "*.cpp")
+file(GLOB PUB_HEADERS "*.hpp")
```

Comment:
```suggestion
file(GLOB PUB_HEADERS "*.h")
```

Reply:
Why not keep the same `set` command - the file names are the same, right?

Reply:
the goal was to have it the same way it is implemented else where in `DAGMC`, not sure it makes a lot of sense for pyne...